python/nano/tutorial/notebook/training/pytorch/accelerate_pytorch_training_multi_instance.ipynb

```diff
@@ -0,0 +1,407 @@
+    "\n",
+    "# subclass TorchNano and override its train method\n",
+    "class MyNano(TorchNano):\n",
+    "    def train(self):\n",
+    "        # Move the code for your custom training loops inside the train method\n",
+    "        model = MyPytorchModule()\n",
+    "        optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.9, weight_decay=5e-4)\n",
+    "        loss_fuc = torch.nn.CrossEntropyLoss()\n",
+    "        train_loader = create_train_dataloader()\n",
+    "\n",
+    "        # call setup method to set up model, optimizer(s),\n",
+    "        # and dataloader(s) for accelerated training\n",
+    "        model, optimizer, train_loader = self.setup(model, optimizer, train_loader)\n",
+    "        num_epochs = 5\n",
+    "\n",
+    "        for epoch in range(num_epochs):\n",
+    "\n",
+    "            model.train()\n",
+    "            train_loss, num = 0, 0\n",
+    "            with tqdm(train_loader, unit=\"batch\") as tepoch:\n",
+    "                for data, target in tepoch:\n",
+    "                    tepoch.set_description(f\"Epoch {epoch}\")\n",
+    "                    optimizer.zero_grad()\n",
+    "                    output = model(data)\n",
+    "                    loss = loss_fuc(output, target)\n",
+    "                    # Replace loss.backward() with self.backward(loss)\n",
+    "                    self.backward(loss)\n",
+    "                    optimizer.step()\n",
+    "                    loss_value = loss.sum()\n",
+    "                    train_loss += loss_value\n",
+    "                    num += 1\n",
+    "                    tepoch.set_postfix(loss=loss_value)\n",
+    "            print(f'Train Epoch: {epoch}, avg_loss: {train_loss / num}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MyNano(num_processes=2).train()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_The detailed definition of_ `MyNano` _can be found in the_ [runnable example](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch/accelerate_pytorch_training_multi_instance.ipynb)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> Two more parameters will be helpful if you want to specify the distributed backend and CPU cores for each process.\n",
+    ">\n",
+    "> * distributed_backend – use which backend in distributed mode, defaults to ‘subprocess’, now available backends are ‘spawn’, ‘subprocess’ and ‘ray’\n",
+    "> * cpu_for_each_process – a list of length `num_processes`, in which each item is a list of CPU indices to specify the cores used by each process, if None, CPU cores will be distributed evenly by all processes, only take effect when `num_processes` > 1"
```

Comment:
I think it may be not necessary to copy the api doc definition here for these two parameters. The important thing here is to mention that for `TorchNano`, `spawn` is supported as a distributed backend, but for `@nano` it does not.

You could add reference to the API doc here for more detailed parameter explanation if you want.

Same for the note box for `@nano`

Reply:
Thanks for your comments! I've modified all of them and rebased my branch.